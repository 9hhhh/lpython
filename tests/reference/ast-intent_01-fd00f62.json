{
    "basename": "ast-intent_01-fd00f62",
    "cmd": "lfortran --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intent_01.f90",
    "infile_hash": "ecbce2d4b01a09bee24ec92ce8be335f75903e746d23a667806bf264",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-intent_01-fd00f62.stdout",
    "stdout_hash": "5f60689e29af29d7799c767135fdca82807a51bf4715b4796d28f78d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}