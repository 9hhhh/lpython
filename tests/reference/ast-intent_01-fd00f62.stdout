(TranslationUnit [(Module dflt_intent [] [] [] [(Subroutine foo [(c) (d)] [] () [] [] [] [(Declaration (AttrType TypeReal [] ()) [] [(c [] [] ()) (d [] [] ()) (e [] [] ()) (g [] [] ())])] [(= 0 e (FuncCallOrArray f [] [(() c () CodimensionExpr)] [])) (= 0 g (FuncCallOrArray f [] [(() d () CodimensionExpr)] []))] [(Function f [(x)] [(AttrType TypeReal [] ())] () () [] [] [] [(Declaration (AttrType TypeReal [] ()) [(AttrIntent In)] [(x [] [] ())])] [(= 0 f (* 2 x)) (Print 0 () [f])] [])])]) (Program main [(Use [] dflt_intent [])] [] [] [(SubroutineCall 0 foo [] [(() (Real "0.0") () CodimensionExpr) (() (Real "2.0") () CodimensionExpr)] [])] [])])
